scanner/mappers/mappers.go

```diff
@@ -592,83 +588,147 @@ var (
 
 // severityAndScores returns the severity and scores information out of a
 // ClairCore vulnerability. The returned information is dependent on the
-// underlying updater. If errors are found in the encoded information, partial
-// values might be returned with an error.
-func severityAndScores(vuln *claircore.Vulnerability, nvdVulns map[string]*nvdschema.CVEAPIJSON20CVEItem) (severityValues, error) {
-	var values severityValues
-	errList := errorhelpers.NewErrorList(fmt.Sprintf("%s: parsing severity and CVSS scores", vuln.Updater))
+// underlying updater.
+func severityAndScores(ctx context.Context, vuln *claircore.Vulnerability, nvdVulns map[string]*nvdschema.CVEAPIJSON20CVEItem) (severityValues, error) {
+	ctx = zlog.ContextWithValues(ctx,
+		"component", "mappers/severityAndScores",
+		"updater", vuln.Updater,
+		"vuln_id", vuln.ID,
+		"vuln_name", vuln.Name,
+	)
+
 	switch {
+	case rhelUpdaterPattern.MatchString(vuln.Updater):
+		return rhelSeverityAndScores(vuln)
```

Comment:
Confirming: RHEL is left empty if not found in V2 as well?

Reply:
In Scanner V2, we don't fall back to NVD if we cannot get this information from Red Hat. I think we tended to just assume Red Hat always had that data